src/twisted/conch/ssh/userauth.py

```diff
@@ -638,83 +656,75 @@ def auth_password(self):
         else:  # returned None, don't do password auth
             return False
 
-    def auth_keyboard_interactive(self):
+    def auth_keyboard_interactive(self) -> bool:
         """
         Try to authenticate with keyboard-interactive authentication.  Send
         the request to the server and return True.
-
-        @rtype: L{bool}
         """
         self._log.debug("authing with keyboard-interactive")
         self.askForAuth(b"keyboard-interactive", NS(b"") + NS(b""))
         return True
 
-    def _cbPassword(self, password):
+    # end of auth_ methods
```

Comment:
Extra markup for end of section

```suggestion
    # end of auth_ methods
    # --------------------
```